src/axom/slic/interface/slic_macros.hpp

```diff
@@ -578,6 +578,35 @@ AXOM_HOST_DEVICE inline blackhole &operator<<(blackhole &bh, T)
                            __LINE__);                 \
   } while(axom::slic::detail::false_value)
 
+/*!
+ * \def SLIC_INFO_TAGGED( msg, tag )
```

Comment:
For now, I have only added one tagged macro, `SLIC_INFO_TAGGED(msg, tag)` for unit testing. It's fairly straightforward to add more, but I figured Axom currently does not have a need for tagged macro variants?